### DISCUSSION TITLE
عدم نمایش JalaliDatePicker در پروژه Asp.net mvc

### DISCUSSION THREAD

**Discussion by bijanjafari** [majidh1/JalaliDatePicker#116] [Question]
باسلام
من هرکاری میکنم نمی تونم در یک پروژه asp.net mvc از JalaliDatePicker استفاده کنم.
یک صفحه ساده که قرار است با کلیک در اینپوت کامپوننت JalaliDatePicker را نمایش دهد.
به پیوست سورس برنامه هم قرار میدم اگر بتونید کمک کنید ممنون میشم.
[WebApplication1.zip](https://github.com/user-attachments/files/17188502/WebApplication1.zip)

**Comment by majidh1** [comment]
سلام شما باید توی layout فایل های js و css را لود کنید و در یک اسکریپت startWatch را فراخوانی کنید. هر input که اتریبیوت data-jdp را داشته باشد دارای datePicker میشود
[Jdp-dotnet.zip](https://github.com/user-attachments/files/17229193/Jdp-dotnet.zip)

**Reply by bijanjafari** [reply]
سپاس مهندس از وقتی که اختصاص دادید
